libs/application/templates/new-primary-school/src/fields/Review/index.tsx

```diff
@@ -0,0 +1,177 @@
+import { newPrimarySchoolMessages } from '../../lib/messages'
+import { getApplicationAnswers } from '../../lib/newPrimarySchoolUtils'
+
+import { AllergiesAndIntolerances } from './review-groups/AllergiesAndIntolerances'
+import { Child } from './review-groups/Child'
+import { Languages } from './review-groups/Languages'
+import { Parents } from './review-groups/Parents'
+import { Photography } from './review-groups/Photography'
+import { ReasonForApplication } from './review-groups/ReasonForApplication'
+import { Relatives } from './review-groups/Relatives'
+import { Siblings } from './review-groups/Siblings'
+import { Support } from './review-groups/Support'
+import { School } from './review-groups/School'
+
+interface ReviewScreenProps {
+  application: Application
+  field: Field & { props?: { editable?: boolean } }
+  goToScreen?: (id: string) => void
+  refetch?: () => void
+  errors?: RecordObject
+  editable?: boolean
+}
+
+export const Review: FC<ReviewScreenProps> = ({
+  application,
+  field,
+  goToScreen,
+  refetch,
+  errors,
+}) => {
+  const { formatMessage } = useLocale()
+  const editable = field.props?.editable ?? false
+  const hasError = (id: string) => get(errors, id) as string
+
+  const { reasonForApplication } = getApplicationAnswers(application.answers)
+
+  const groupHasNoErrors = (ids: string[]) =>
+    ids.every((id) => !has(errors, id))
+
+  const childProps = {
+    application,
+    editable,
+    groupHasNoErrors,
+    hasError,
+    goToScreen,
+  }
+
+  const { state } = application
+
+  const [submitApplication, { loading: loadingSubmit }] = useMutation(
+    SUBMIT_APPLICATION,
+    {
+      onError: (e) => handleServerError(e, formatMessage),
+    },
+  )
+
+  const handleSubmit = async (event: string) => {
+    const res = await submitApplication({
+      variables: {
+        input: {
+          id: application.id,
+          event,
+          answers: application.answers,
+        },
+      },
+    })
+
+    if (res?.data) {
+      // Takes them to the next state (which loads the relevant form)
+      refetch?.()
+    }
+  }
+
+  return (
+    <>
+      {state === `${States.DRAFT}` && (
+        <Box display="flex" justifyContent="spaceBetween">
+          <Box>
+            <Box marginBottom={2}>
+              <Text variant="h2">
+                {formatMessage(newPrimarySchoolMessages.overview.sectionTitle)}
+              </Text>
+            </Box>
+            <Box marginBottom={10}>
+              <Text variant="default">
+                {formatMessage(
+                  newPrimarySchoolMessages.overview.overviewDescription,
+                )}
+              </Text>
+            </Box>
+          </Box>
+          <Box>
+            <Button
+              variant="utility"
+              icon="print"
+              onClick={(e) => {
+                e.preventDefault()
+                window.print()
+              }}
+            />
```

Comment:
This button should have `type="button", otherwise it defaults to `submit`, which we don't want inside the form.

Reply:
Doesn't it default to button?

<img width="327" alt="image" src="https://github.com/island-is/island.is/assets/25203781/e6764d6d-cfc8-470d-ae17-5898ccc30a77">
<br>
<img width="175" alt="image" src="https://github.com/island-is/island.is/assets/25203781/90a7c296-ef14-4a91-8fda-a410307d0b40">

Reply:
Oh nice TIL!